src/v/config/node_config.cc

```diff
@@ -155,6 +155,12 @@ node_config::node_config() noexcept
       "Enables log messages for allocations greater than the given size.",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , debug_bundle_name(
+      *this,
+      "debug_bundle_name",
+      "The file name for the debug bundle zip file (default: debug-bundle.zip)",
+      {.visibility = visibility::user},
+      "debug-bundle.zip")
```

Comment:
I think this should require some validation when this configuration item is changed so we know we can write to the directory this file is located in.

Here, `debug-bundle.zip` is going to end up up in the directory Redpanda is running in (not necessarily the directory `redpanda` is installed in).  So this could be in `/` which is normally a `root:root` owned directory.

Reply:
good point, I'll make that change.

Reply:
I'm making the default locale `/tmp/debug-bundle.zip`

Reply:
Spoke offline with @michael-redpanda ,
Instead of adding a config op for the filename, we will add a config op for the directory which RP will write debug archives.